include/options/business/media-support.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+namespace WP_Syntex\Polylang\Options\Business;
+
+use WP_Error;
+use WP_Syntex\Polylang\Options\Primitive\Abstract_Boolean;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class defining the "Translate media" boolean option.
+ *
+ * @since 3.7
+ */
+class Media_Support extends Abstract_Boolean {
+	/**
+	 * Constructor.
+	 *
+	 * @since 3.7
+	 *
+	 * @param string $key   Option key.
+	 * @param mixed  $value Optional. Option value.
+	 *
+	 * @phpstan-param non-falsy-string $key
+	 */
+	public function __construct( string $key, $value = null ) {
+		parent::__construct( $key, $value, false );
+	}
```

Comment:
By doing this, child classes can't change the default value anymore, since this constructor has now 2 parameters instead of 3: in child classes we can't call `parent::__construct( $key, $value, true );` (there is no 3rd parameter).
As an alternative, I suggest to not pass the default value as an argument, but set the property directly in (child) classes.

Reply:
Done in [e6e1420](https://github.com/polylang/polylang/pull/1481/commits/e6e1420341cf9c6815a6f55899280b504bc119a3).